DSCResources/MSFT_xADOrganizationalUnit/MSFT_xADOrganizationalUnit.schema.mof

```diff
@@ -0,0 +1,12 @@
+[ClassVersion("1.0.0.0"), FriendlyName("xADOrganizationalUnit")] 
+class MSFT_xADOrganizationalUnit : OMI_BaseResource
+{
+    [Key, Description("The name of OU")] string Name;
+    [Key, Description("Specifies the X500 path of the OU or container where the new object is created")] string Path;
+   
+    [Write, ValueMap{"Present", "Absent"}, Values{"Present", "Absent"}] string Ensure;
+    [Write, EmbeddedInstance("MSFT_Credential"),Description("The credential to be used to perform the operation on Active Directory")] string Credential;
+    [Write, ValueMap{"Yes", "No"}, Values{"Yes", "No"},Description("Defaults to Yes")] string ProtectedFromAccidentalDeletion;
```

Comment:
I think we should convert this to a boolean, unless there is a really good reason to leave it yes/no.